tests/e2e/install_suite_test.go

```diff
@@ -1,37 +1,65 @@
 package e2e
 
 import (
+	"flag"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 // var _ = BeforeSuite(func() {
-// 	err := installDefaultVelero()
-// 	Expect(err).ToNot(HaveOccurred())
+// 	prefix := "ts-" // To create individual instance per spec
+// 	flag.Parse()
+// 	s3Data := decodeJson(getJsonData(s3BuckerFilePath)) // Might need to change this later on to create s3 for each tests
+// 	s3Bucket = s3Data["velero-bucket-name"].(string)
+// 	credData := getCredsData(cloud)
+// 	err := createSecret(credData, namespace, credSecretRef)
+// 	Expect(err).NotTo(HaveOccurred())
 
-// 	//err := createOADPTestNamespace()
-// 	//Expect(err).NotTo(HaveOccurred())
+// 	namespace = prefix + namespace
+// 	s3Bucket = prefix + s3Bucket
+// 	credSecretRef = prefix + credSecretRef
+// 	instanceName = prefix + instanceName
+// 	err = createOADPTestNamespace(namespace)
+// 	Expect(err).NotTo(HaveOccurred())
 // 	// Check that OADP operator is installed in test namespace
 // })
 
 // var _ = AfterSuite(func() {
-// 	err := uninstallVelero()
-// 	Expect(err).ToNot(HaveOccurred())
+// 	err := deleteSecret(namespace, credSecretRef)
+// 	Expect(err).NotTo(HaveOccurred())
+// 	err = deleteOADPTestNamespace(namespace)
+// 	Expect(err).NotTo(HaveOccurred())
 // })
 
 // Check Velero is deleted
 // TODO: Check test namespace is deleted
 // TODO: Check secret is deleted
-//})
+// })
 
 var _ = Describe("The default Velero custom resource", func() {
 	var _ = BeforeEach(func() {
-		err := installDefaultVelero()
+		prefix := "ts-" // To create individual instance per spec
+		flag.Parse()
+		s3Data := decodeJson(getJsonData(s3BuckerFilePath)) // Might need to change this later on to create s3 for each tests
+		s3Bucket = s3Data["velero-bucket-name"].(string)
```

Comment:
I'm curious is this actually going to work? My understanding was our latest version of the secret encodes the JSON in a file under `velero-credentials`. So to load the bucket, I thought we had to do something like:
```
data := decodeJson(getJsonData("/var/run/velero-credentials"))
s3Bucket := data["velero-bucket-name"]
```

Not totally sure though
```

Reply:
So, i have divided the logic for getting the aws credentials. now two files will be created , one containing s3 bucket and other necessary details. and the other aws credentials created just like how we want to pass when installing / creating oc secret.

Reply:
Ahh I see. Nice makes sense, thanks!